modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -514,11 +573,25 @@ def map_func(left, right=right, kwargs=kwargs):
     def reindex(self, axis, labels, **kwargs):
         new_index = self.index if axis else labels
         new_columns = labels if axis else self.columns
+        actual_index = None
+        if "_reset_index" in kwargs:
+            actual_index = kwargs.pop("_reset_index")
+
+        def _reindex(df):
+            kwargs["copy"] = False
```

Comment:
this changes kwargs, which is probably benign but not good practice. can you just pass copy to df.reindex